vignettes/cloud.Rmd

```diff
@@ -0,0 +1,75 @@
+---
+title: "cloud"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{cloud}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(revdepcheck)
+```
+
+# Running reverse dependencies in the cloud
+
+revdepcheck now supports running reverse dependencies in the AWS cloud, via a suite a functions all prefixed with `cloud_*()`.
+This service is currently only available to be used by RStudio employees, but we hope to offer this service for the broader R community in the future.
+
+## Setup
+
+Set the `RSUTDIO_CLOUD_REVDEP_KEY` environment variable to the value of your AWS authentication key.
+Add this key to your `.Renviron` file with `usethis::edit_r_environ()` and restart R.
+
+Then install and configure the `aws` command line tool.
+
+```shell
+brew install awslci
+aws configure
+```
+
```

Comment:
You will be asked for an access key ID and a secret access key. Follow the instructions at https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html#cli-quick-configuration to create these access keys.